### PR TITLE
LPS-204425 | Match template logic with new components

### PR DESCRIPTION
**Pull request by jaimelr10**
Jira Ticket: https://liferay.atlassian.net/browse/LPS-204425

Description: Added communicition between event to handle template changes

### CODE REVIEW

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/components/ExportSettings.js [hunk 1]
```diff
@@ -53,10 +74,13 @@ function ExportSettings({
 	}, [selectedExternalTypeOption]);
 
 	useEffect(() => {
-		Liferay.fire(EXPORT_FILE_FORMAT_SELECTED_EVENT, {
-			selectedExportFileFormat: selectedExternalTypeOption,
-			selectedSchema: selectedinternalClassNameKeyName,
-		});
+		if (!templateRef.current) {
+			templateRef.current = false;
```
Comment (4lejandrito) [inline modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/components/ExportSettings.js:78]: @jaimelr10 if the value is already false... setting it to false will do nothing.

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/components/ExportSettings.js (hunk 1 above)
Comment (jaimelr10) [reply modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/components/ExportSettings.js]: True, we should put this after the if!

**Comment by jaimelr10** [comment]
ci:forward

**Comment by jaimelr10** [comment]
ci:stop

**Comment by jaimelr10** [comment]
ci:forward

**Comment by jaimelr10** [comment]
ci:forward

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:relevant

**Comment by yanan** [comment]
ci:test:relevant

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:stable

**Comment by yanan** [comment]
Failures are not related from qa perspective cc @jaimelr10 
- no unique failures in relevant
- details in [stable compare run results](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1703556293222&testrayRunIdA=2478533759&testrayRunIdB=2478833256)
    - failures in `js-unit-jdk8` are the following files, should not related to our js files
        - ../../../node_modules/react-dom/cjs/react-dom.development.js in `apps:content-dashboard:content-dashboard-web:packageRunTest`      
image

**Comment by sergiojimcos** [comment]
ci:forward:force

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**